Rollbar/RollbarConfiguration.m

```diff
@@ -91,10 +94,22 @@ - (void)setServerHost:(NSString *)host root:(NSString*)root branch:(NSString*)br
     [self save];
 }
 
-- (void)setPayloadModificationBlock:(void (^)(NSDictionary*))payloadModificationBlock {
+- (void)setPayloadModificationBlock:(void (^)(NSMutableDictionary*))payloadModificationBlock {
     self.payloadModification = payloadModificationBlock;
 }
 
+- (void)setCheckIgnoreBlock:(BOOL (^)(NSDictionary *))checkIgnoreBlock {
+    self.checkIgnore = checkIgnoreBlock;
+}
+
+- (void)addScrubField:(NSString *)field {
+    [self.scrubFields setObject:@"1" forKey:field];
```

Comment:
You could use an `NSMutableSet` instead of an `NSMutableDictionary` based on how this is used.

Reply:
Changed to set instead of dictionary